php.ini-production

```diff
@@ -819,6 +819,11 @@ enable_dl = Off
 ; https://php.net/fastcgi.impersonate
 ;fastcgi.impersonate = 1
 
+; Prevent decoding of SCRIPT_FILENAME if Apache ProxyPass or ProxyPassMatch is
+; used. This should not be used unless the script file paths are saved already
+; encoded in the file system.
```

Comment:
Like the other INI settings, maybe it's a good idea to explicitly spell out that it's disabled by default?
(And then same thing for the development INI file)

Reply:
Done